packages/block-library/src/navigation/edit/overlay-menu-preview.js

```diff
@@ -20,7 +20,7 @@ export default function OverlayMenuPreview( { setAttributes, hasIcon, icon } ) {
 				__nextHasNoMarginBottom
 				label={ __( 'Show icon button' ) }
 				help={ __(
-					'Configure the visual appearance of the button opening the overlay menu.'
+					'Configure the visual appearance of the button opening and closing the overlay menu.'
```

Comment:
```suggestion
					'Configure the visual appearance of the button that toggles the overlay menu.'
```

Just an idea to reduce verbosity.